### PR TITLE
Work flow

### PR DESCRIPTION
**Pull request by jitwit**

### CODE REVIEW

**Comment by jitwit** [comment]
Still need to update documentation for running tests instructions

File: exercises/atbash-cipher/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+solution := 
+
+chez := scheme
+guile := guile
+
+help :
+	echo "Test your solutions from the command line by calling make *, where * is the name of your scheme."
```
Comment (guygastineau) [inline exercises/atbash-cipher/Makefile]: I ti probably safer top use single quotes here.  I guess that the `make` probably doesn't do shell expansion first, but if somebody tries to just copy this line for some reason and run it in their terminal, then they will get all of the contents of their current directory everytime there is a `*` I think.

**Comment by guygastineau** [comment]
This is good work.  It is also a lot ;)

I spot checked what is going on, but I would also like to dig in further after I eat some protein :rofl: 

I think there is a great opportunity to add some further refactoring to the tests here, and it might be beneficial to do all of that in one go.

There are various architectures that could be used now that there is a Makefile.  For example:
```
solution.scm
Makefile
tests/test.scm
tests/test.sls
```

@jitwit how do feel about putting this in `feature` or another base branch for a little while, so we can make some more substantial changes before launching? It would seem advantageous to me if a couple iterations of upgrades to our testing framework happened in isolation before getting sent out to students.

Let me know if you are down to handle it this way.

PS. I am not glued to the example architecture. It was just a convenient example :smile:

**Comment by jitwit** [comment]
I'd prefer to incorporate this sooner rather than later just because it makes adding problems a lot more natural. It also makes for better student development. 

It would be great to start another branch to explore more complicated exercise architectures as you say though!

File: exercises/atbash-cipher/Makefile (hunk 1 above)
Comment (jitwit) [reply exercises/atbash-cipher/Makefile]: haha good to know!

**Pull request merged**